content/docs/get-started/install/migrating-3.0.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Migrating to Pulumi 3.0
+meta_desc: This page provides instructions for migrating to Pulumi 3.0
+no_on_this_page: true
+---
+
+## Migrating to 3.0
+
+Upgrading to 3.0 is simple. First, you will [install the 3.0 CLI]({{< relref "/docs/get-started/install" >}}). Then, update each of your Pulumi programs to utilize the new SDK.
+
+## Changes to the CLI Behaviour
```

Comment:
```suggestion
## CLI behavior changes in Pulumi 3.0
```

Do we have a standard style guide/point of view on American vs. British spellings?

Reply:
American - I just speak the correct version of English ;) J/k!